examples/java/pom.xml

```diff
@@ -48,6 +48,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <trimStackTrace>false</trimStackTrace>
```

Comment:
It is currently set at the top-level `pom.xml` here: https://github.com/apache/incubator-beam/blob/983d467a798458e7ee4f73cd8263e6044903bd0e/pom.xml#L810

Does this configuration section overwrite it?